src/modules/gfx_voronoi.c

```diff
@@ -0,0 +1,357 @@
+// Voronoi Diagramm
+//
+// This effect is inspired by https://github.com/tsoding/
+//
+// Copyright (c) 2022, Olaf "Brolf" Pichler <brolf@magheute.net>
+//
+// Permission to use, copy, modify, and/or distribute this software for any
+// purpose with or without fee is hereby granted, provided that the above
+// copyright notice and this permission notice appear in all copies.
+//
+// THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+// WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+// MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+// ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+// WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+// ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+// OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+#include <types.h>
+#include <matrix.h>
+#include <timers.h>
+#include <stddef.h>
+#include <random.h>
+#include <string.h>
+#include <stdlib.h>
+#include <assert.h>
+#include <mathey.h>
+#include <stdio.h>
+
+#define FPS 60
+#define FRAMETIME (T_SECOND / FPS)
+#define FRAMES (TIME_LONG * FPS)
+
+static int modno;
+static unsigned int frame;
+static oscore_time nexttick;
+static float delta0;
+
+static uint16_t xmax;
+static uint16_t ymax;
+
+
+static const RGB white = RGB(255, 255, 255);
+static const RGB black = RGB(0,0,0);
+
+static enum NormType{
+    MULTIPLICATE = 0,
+    SUM = 1,
+    EUKLID = 2,
+    P3 = 3,
+    P4 = 4,
+    P5 = 5,
+    P6 = 6,
+    MAX = 7,
+} norm_type = MULTIPLICATE;
+
+static const norm_rand_min = 0;
+static const norm_rand_max = 7;
+static uint16_t choose_random_norm = 0;
+
+#define P_MAX 20
+static struct Point {
+	float x;
+	float y;
+    float vx;
+    float vy;
```

Comment:
Mismatched indentation here. Maybe convert everything to tabs and check it's all sane? (Or spaces if you prefer that...)